.gitignore

```diff
@@ -53,8 +53,10 @@ build/builtin-actors/v*
 build/builtin-actors/*.car
 
 dist/
+# deprecated
 darwin_amd64_v1/
 darwin_arm64/
+# deprecated
 darwin_all/
```

Comment:
Q: Do we want to keep publishing under the `darwin_all` name even though it would contain `darwin_arm64` only now?

Reply:
I think we be consistent with linux and just have `darwin_arm64` and not have an `all`.

<img width="1085" height="384" alt="Image" src="https://github.com/user-attachments/assets/89741d2f-15fe-4b30-a485-7b2431f85974" />